app/src/main/res/values/strings.xml

```diff
@@ -784,6 +784,8 @@ Upload your first media by tapping on the add button.</string>
   <string name="storage_permissions_denied">Storage Permissions Denied</string>
   <string name="unable_to_share_upload_item">Unable to share this item</string>
   <string name="permissions_are_required_for_functionality">Permissions are required for functionality</string>
+  <string name="learn_how_to_write_better_description">Learn how to write better description</string>
```

Comment:
I would suggest: `Learn how to write a useful description`

Reply:
Thank you for suggestion, I made the suggested changes in commit https://github.com/commons-app/apps-android-commons/pull/5415/commits/0dc6b1123353005fbcc881d6c89d6abee0d008a0